tests/e2e/standard/test_fetch_tables.py

```diff
@@ -155,3 +156,25 @@ def get_model_versions_as_rows(**kwargs):
             return container.fetch_model_versions_table(**kwargs).to_rows()
 
         self._test_fetch_from_container(init_run, get_model_versions_as_rows)
+
+    @pytest.mark.parametrize(
+        "state_active,state_inactive", [("active", "inactive"), ("Active", "Inactive"), ("aCTive", "inAcTiVE")]
```

Comment:
Same, let's just try to filter from API with some "new" value for state in e2e. In unittests -> Check that is case insensitive.